Find a functionally equivalent code:android.hardware.SystemSensorManager.unregisterListenerImpl:COMMENT
Method Modifier: protected   hidden      
Comment:/**
 * @hide
 */

@@ -1,6 +1,6 @@
 {
     // Trigger Sensors should use the cancelTriggerSensor call.
-    if (sensor != null && Sensor.getReportingMode(sensor) == Sensor.REPORTING_MODE_ONE_SHOT) {
+    if (sensor != null && sensor.getReportingMode() == Sensor.REPORTING_MODE_ONE_SHOT) {
         return;
     }
     synchronized (mSensorListeners) {

